### DISCUSSION TITLE
Should GitHub organizations feature only be used for open-source? Can it be used for collaborating work only?

### DISCUSSION THREAD

**Discussion by anirudhvee** [community/community#23749] [New to GitHub]
<p>Should GitHub organizations feature only be used for open-source? Can it be used for collaborating work only?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/anirudhvee">@anirudhvee</a>, Welcome to the community! We are happy you are here. GitHub free is for all organizations not only open source. For more information about creating organization check out: <a href="https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/creating-a-new-organization-from-scratch">https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/creating-a-new-organization-from-scratch</a></p>

**Comment by anirudhvee** [comment]
<p>Ok Thanks!</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/6/677f06bdf711468707faf594c72c7993f90d2513.png" data-download-href="/uploads/short-url/eLzkAGZovUCazoJL65K1uagmWFJ.png?dl=1" title="0029ED679A0D427C91CE5298C857643B.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/50367124/181079224-73151800-50a9-4983-a417-87e40737fd30.png" alt="0029ED679A0D427C91CE5298C857643B.png" data-base62-sha1="eLzkAGZovUCazoJL65K1uagmWFJ" width="690" height="0" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/6/677f06bdf711468707faf594c72c7993f90d2513_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">0029ED679A0D427C91CE5298C857643B.png</span><span class="informations">708×1 83 Bytes</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
